mne_bids_pipeline/steps/preprocessing/_04_frequency_filter.py

```diff
@@ -190,6 +193,8 @@ def filter_data(
 
     out_files[in_key] = bids_path_in.copy().update(
         root=cfg.deriv_root,
+        subject=subject,  # save under subject's directory so all files are there
+        session=session,
```

Comment:
Previously the `filt` step for SSS processing would output to `sub-01/*` type paths, whereas if SSS was off it would output to `sub-emptyroom/*` type paths. This unifies it so everything lands in subject-specific paths. This means that after the `filt` step (e.g., during regression, SSP, and ICA) we can assume all files for each subject come from that subject's path rather than potentially `sub-emptyroom`. This may mean a few redundant files but simplifies logic and should help with #831.